react/src/css/App.css

```diff
@@ -179,33 +174,32 @@
 }
 
 .check-run-button {
-   border: 1px solid lightgreen;
+   border: 1px solid var(--box-fg-darken);
    border-radius: 12px;
    padding-left: 8pt;
    padding-top: 2pt;
    padding-bottom: 3pt;
    padding-right: 12pt;
-   background: #4F7942;
+   background: var(--box-fg);
    color: #FFFFFF;
    cursor: pointer;
    user-select: none;
 }
 .check-run-button:hover {
     color: yellow;
-    background: darkgreen;
+    background: var(--box-fg);
 }
 .check-run-button:active {
-    color: red;
+    color: darkred !important;
```

Comment:
I've never wrapped my head around what `!important` does. I've seen people argue it has the opposite effect from importance, but I confess I don't reason well about it. I'm curious what this is supposed to do to it, though.